### PR TITLE
Including OPT_OUT feature for user-agent telemetry

### PR DESCRIPTION
**Pull request by leonardonakagawa**
This commit includes new feature for Python use-cases where user can opt-out User-Agent tracking

Just apiutils.py file was changed in 

Data Kiosk
Pricing
Solicitations
Shipping V2

### CODE REVIEW

File: use-cases/pricing/code/python/src/utils/api_utils.py [hunk 1]
```diff
@@ -40,6 +41,11 @@ def _get_app_credentials(self):
             raise e
         else:
             return s_dict
+   
+    def _set_useragent(self, client):
+        print('Setting user agent')
```
Comment (weilinggu) [inline use-cases/pricing/code/python/src/utils/api_utils.py:46]: @leonardonakagawa Should we move the print after if?

**Review by weilinggu — APPROVED** [review]
Can you move the print statement? Otherwise it looks good

File: use-cases/pricing/code/python/src/utils/api_utils.py (hunk 1 above)
Comment (rodrifed) [reply use-cases/pricing/code/python/src/utils/api_utils.py:46]: +1

**Comment by rodrifed** [comment]
Could you please update the README files with instructions on how to opt out of the custom user agent?

**Pull request merged**